resource/pipeline.yaml

```diff
@@ -13,6 +13,13 @@ jobs:
   - task: webapp-source
     file: common-tasks/extract-release-tarball.yaml
 
+  - task: deploy-params
```

Comment:
Is this `deploy-params` just the name given to this task? If it can be anything, maybe we can change it to something like `parse-deploy-params` or just `parse-params`, as my understanding of what the `parse-deploy-file.yaml` resource is doing right?

Reply:
This should match the name of the output directory. It's either this, or load the results from `parse-deploy-params/override.yaml`, which I think is less good.

Reply:
👍